src/base/filesystemwatcher.cpp

```diff
@@ -164,8 +157,7 @@ void FileSystemWatcher::processPartialTorrents()
 
     // Stop the partial timer if necessary
     if (m_partialTorrents.empty()) {
-        m_partialTorrentTimer->stop();
-        m_partialTorrentTimer->deleteLater();
+        delete m_partialTorrentTimer;
```

Comment:
You should not delete sender from slot! deleteLater() was appropriate way.

Reply:
oops, will revert.